src/components/Calculator.js

```diff
@@ -1,42 +1,64 @@
 import React from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate.js';
 
 class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    this.handleClickEvent = this.handleClickEvent.bind(this);
+  }
+
+  handleClickEvent(event) {
+    const buttonName = event.target.id;
+    const obj = calculate(this.state, buttonName);
+    const { total, next, operation } = obj;
+    this.setState({
+      total,
+      next,
+      operation,
+    });
+  }
+
   render() {
     return (
             <div className='wrapper'>
                 <div className='calculator'>
                     <ul className='row-1'>
-                        <li className='result'>0</li>
+                        <li className='result'>{this.state.operation !== null ? this.state.next : this.state.total || 0}</li>
                     </ul>
                     <ul className='row row-2'>
-                        <li className='calc-items'>AC</li>
-                        <li className='calc-items'>+/-</li>
-                        <li className='calc-items'>%</li>
-                        <li className='calc-items calc-operation'>/</li>
+                        <li className='calc-items' id='AC' onClick={this.handleClickEvent}>AC</li>
+                        <li className='calc-items' id='+/-' onClick={this.handleClickEvent}>+/-</li>
+                        <li className='calc-items' id='%' onClick={this.handleClickEvent}>%</li>
+                        <li className='calc-items calc-operation' id='/' onClick={this.handleClickEvent}>/</li>
                     </ul>
```

Comment:
- [ ] Please make sure that the "division" process works as intended. Currently, if the user tries to perform a division process, it produces the following error (See attached below).
![image](https://user-images.githubusercontent.com/62290545/206519356-60f41019-3079-4972-8531-23f4de452983.png)

- [HINT] If you investigated the `operate.js` file, you'll see that it uses the `÷` operator for the division and not the `/`.